src/state/lists/reducer.ts

```diff
@@ -66,11 +68,10 @@ export default createReducer(initialState, (builder) =>
         if (upgradeType === VersionUpgrade.NONE) return
         if (loadingRequestId === null || loadingRequestId === requestId) {
           state.byUrl[url] = {
-            ...state.byUrl[url],
```

Comment:
doesn't removing `...state.byUrl[url]` change the behavior? Can't remember if we're using imer in the background

Reply:
imer? According to the typing, this should not change the behavior because all of the properties from state.byUrl[url] are overwritten on the next 4 lines.

Reply:
ah okay makes sense

was referring to https://github.com/immerjs/immer